index.js

```diff
@@ -103,20 +103,31 @@ const packageLockFilename = path.join(
 )
 
 const useYarn = () => fs.existsSync(yarnFilename)
-
 const usePnpm = () => fs.existsSync(pnpmLockFilename)
+const useNpm = () => fs.existsSync(packageLockFilename)
 
 const lockHash = () => {
   const lockFilename = useYarn()
```

Comment:
Just a style thing, but since we've got more than two conditions, should we use a good old if statement?

```js
if (useYarn()) {
  return ...
}

// ... more if statements 

return noLockFile()

Reply:
If I re-wrote this section I would want to put in more debug statements about which lockfiles were found, as there is another condition where there is more than one lockfile found. I decided not to try to catch this condition though. `yarn` will warn about this. Other package managers don't mind.

So there is scope for improving this area of the code, however my tendency right now would be to leave the PR in its current state, since it is tested and working (and there are plenty of other things which still need addressing!).

Reply:
I don't feel *strongly* about it - if it works, it works -- but always good to make things incrementally better.

Happy to approve for now, will loop in @AtofStryker for another set of :eyes:.